katas/Palindrom/solutions/JLintz/testcases.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+'''
+@author Julian Lintz
+@brief Testcases for solution to kata 'palindrom'
+'''
+import unittest
+import random
+import string
+import re
+from palindrom import isPalindrom
+
+
+class PalindromTestCases(unittest.TestCase):
```

Comment:
probably a `setUp()` would be useful for the tests. The examples from the presentation could be regarded as fixture.